source/ed/connectors/gtfs_parser.h

```diff
@@ -106,6 +107,9 @@ struct GtfsData {
     //for gtfs we group the comments together, so the key is the comment and the value is the id of the comment
     std::unordered_map<std::string, std::string> comments_id_map;
 
+    // Store lines linked for each group to avoid duplicates
+    std::unordered_map<std::string, std::vector<std::string>> linked_lines_by_line_group_uri;
```

Comment:
It's only to store uris of lines of each group. I don't really mind duplicate inside the vector, I just use "find" before creating a link between the group and the line.
And since I read http://lafstern.org/matt/col1.pdf I try to not use them too much =P.